framework/src/main/java/org/checkerframework/framework/util/element/ElementAnnotationUtil.java

```diff
@@ -84,8 +84,10 @@ public static void applyAllElementAnnotations(
      * @param type the type to annotate
      * @param annotations the annotations to add
      */
-    static void addAnnotationsFromElement(
+    static void addDeclarationAnnotationsFromElement(
```

Comment:
I'll probably merge this change separately, but I think this rename should help clarify what's happening here.

Reply:
Yes, this is a better name.